test/acceptance/properties.json

```diff
@@ -17,9 +17,9 @@
   },
   "deploy_data": {
     "host": "localhost",
-    "ssh_port": "22",
-    "user": "root",
-    "password": "root",
-    "bin_path": "/usr/bin/contextBroker"
+    "ssh_port": "",
```

Comment:
It's a bit weird having defined properties that you don't use at the end... Should this be mentioned in the README.md for acceptance tests?